grpc/src/main/java/com/linecorp/armeria/client/grpc/GrpcWebUtil.java

```diff
@@ -75,6 +76,9 @@
      *                })))
      *        .build(MyGrpcStub.class);
      * }</pre>
+     *
+     * @throws RequestLogAvailabilityException if the specified {@link RequestLogProperty#SCHEME} is not
+     *                                         available yet.
```

Comment:
A user did not specify `RequestLogProperty`, so please rephrase it.

Reply:
Oops fixed. 😉